### PR TITLE
PCX-861 GraphQL Analytics API tokens

### PR DESCRIPTION
**Pull request by aaron-ruby-df**
## User story

Update the /graphql-api/ documentation with a walkthrough on how to configure CloudflareAPI tokens so that users can use them securely authenticate to the GraphQL Analytics API.

### CODE REVIEW

**Review by tlozoot — APPROVED** [review]
LGTM. Would be good to have feedback from Tom on this as well.

**Comment by tlozoot** [comment]
We should also wait to merge this until Tom is happy that the open bugs have been resolved.

File: products/analytics/src/content/graphql-api/getting-started/authentication/index.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  <thead>
+      <tr>
+          <th>Authentication Method</th>
+          <th>Description</th>
+      </tr>
+  </thead>
+  <tbody>
+      <tr>
+        <td><a href='https://developers.cloudflare.com/api/keys'>API Keys</a></td>
+        <td><p>Unique to each Cloudflare user and used only for authentication. API keys not authorize access to accounts or zones.</p>
```
Comment (twalwyn) [inline products/analytics/src/content/graphql-api/getting-started/authentication/index.md]:
```suggestion
        <td><p>Unique to each Cloudflare user and used only for authentication. API keys do not authorize access to accounts or zones.</p>
```

File: products/analytics/src/content/graphql-api/index.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ title: GraphQL Analytics API
 order: 1
 ---
 
-# GraphQL Analytics API
+# GraphQl Analystics API
```
Comment (twalwyn) [inline products/analytics/src/content/graphql-api/index.md]:
```suggestion
# GraphQl Analytics API
```

File: products/analytics/src/content/graphql-api/getting-started/authentication/api-token-auth.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
+---
```
Comment (twalwyn) [inline products/analytics/src/content/graphql-api/getting-started/authentication/api-token-auth.md:1]: My understanding of this doc is that it describes how to create an analytics-scoped API token. Do we also want to include information about how to use the token to query the GraphQL API?

We have this information for "Authenticate with a Cloudflare API key". In that doc we provide examples of how to run queries with GraphiQL.

**Review by twalwyn — COMMENTED** [review]
Thanks! A few small things, but in general LGTM.

File: products/analytics/src/content/graphql-api/getting-started/authentication/api-token-auth.md (hunk 3 above)
Comment (paulo-docforce) [reply products/analytics/src/content/graphql-api/getting-started/authentication/api-token-auth.md:1]: Thanks for pointing out those bugs, we are going to fix them with a final copy-edit pass. 

We would suggest that, if there are no other open bugs and the content is valid, you should publish it and then file another ticket for a tutorial on how to use the token to query the GraphQL API.

This way we avoid holding back this content and also keep track of the next iterations.

What do you say?

**Comment by paulo-docforce** [comment]
Thanks for pointing out those bugs, we are going to fix them with a final copy-edit pass. 

We would suggest that, if there are no other open bugs and the content is valid, you should publish it and then file another ticket for a tutorial on how to use the token to query the GraphQL API.

This way we avoid holding back this content and also keep track of the next iterations.

What do you say?

**Comment by paulo-docforce** [comment]
Comments from your review have been implemented and PCX-877 is now ready to go.

**Comment by twalwyn** [comment]
> Thanks for pointing out those bugs, we are going to fix them with a final copy-edit pass.
> 
> We would suggest that, if there are no other open bugs and the content is valid, you should publish it and then file another ticket for a tutorial on how to use the token to query the GraphQL API.
> 
> This way we avoid holding back this content and also keep track of the next iterations.
> 
> What do you say?

Sounds good to me.

**Review by twalwyn — APPROVED** [review]

**Comment by soheiokamoto** [comment]
Merged.
I still think `GraphQl` letter case should be fixed, though.

**Comment by paulo-docforce** [comment]
> Merged.
> I still think `GraphQl` letter case should be fixed, though.

Agreed @soheiokamoto, we'll make sure the letter case is consistent throughout the product area.

This issue is now being handled under [PCX-1069](https://jira.cfops.it/browse/PCX-1069).

**Pull request merged**